<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteDialog</class>
 <widget class="QDialog" name="DeleteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Confirm Delete</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>159</width>
     <height>44</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::No|QDialogButtonBox::Yes</set>
   </property>
  </widget>
  <widget class="QLabel" name="iconlabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>35</y>
     <width>71</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="messageLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>291</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Are you sure you want to delete &quot;filename.txt&quot;?

This action cannot be undone.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="iconLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="pixmap">
    <pixmap>:/qt-project.org/styles/commonstyle/images/standardbutton-question-32.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignTop</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
